src/main/java/appeng/me/storage/BasicCellInventory.java

```diff
@@ -111,23 +134,23 @@ public IAEItemStack injectItems( final IAEItemStack input, final Actionable mode
 			return null;
 		}
 
-		if( this.cellType.isBlackListed( this.i, input ) )
+		if( this.cellType.isBlackListed( this.getItemStack(), input ) )
 		{
 			return input;
 		}
 		// This is slightly hacky as it expects a read-only access, but fine for now.
 		// TODO: Guarantee a read-only access. E.g. provide an isEmpty() method and ensure CellInventory does not write
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)